app/experimenter/experiments/models.py

```diff
@@ -267,8 +268,39 @@ def bugzilla_url(self):
 
     @property
     def monitoring_dashboard_url(self):
-        if self.is_begun and self.normandy_slug:
-            return settings.MONITORING_URL.format(slug=self.normandy_slug)
+        start_date = ""
+        end_date = ""
+
+        if (
+            self.status == self.STATUS_LIVE
```

Comment:
Ah sorry to be a pain about this but I'd really rather you use the aggregate status aliases, `is_begun` in this case.  That way if we add/change statuses in the future and we can update the aliases without needing to update the call sites.